m4_sessions/1801-inning/group_3.md

```diff
@@ -1,13 +1,19 @@
 Group 3
 
 
-Group Members:
+Group Members: Maria St. Louis-Sanchez, Katie Scruggs, Julie Hawkins, Jason Hughes
 
-Name of Session:
+Name of Session: Relaxation in Practice
 
-Session Description (Why should students come to this session):
+Session Description (Why should students come to this session):  Students who come to this session will learn and practice techniques to help them relax during stressful times. They will also reflect on what strategies will work for them (or already work for them), and how to recognize when it is time to use these techniques and step away from the computer.
 
-Session Outcomes (What will students learn/leave with?):
+Session Outcomes (What will students learn/leave with?): Students will leave with an awareness of how important it is to make time to do activities that relax you. They will also have a clear list of activities they're interested in, and a Slack channel set up to share ideas and for support.
 
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
 
+Intro (10 min)- Lights will be off and calming music will be playing. Welcome to the session, facilitators will introduce ourselves. Icebreaker = what do you already do to relax? what makes you feel like you?
+
+Meditation Session (5)- We will have a small mindfulness meditation session via YouTube Video.
```

Comment:
Very nice. Do you have a slack channel?  If so, can you add me to it?  Might be cool to open up the slack channel to the entire school after the session so people could share their best relaxation resources.